### PR TITLE
Update config files to prepare sample app for AKS migration

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-840

### Change description ###
Update config, clean up properties and bump charts version. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-sample-app/values.aat.template.yaml [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-  keyVaults:
-    "s2s":
-      resourceGroup: rpe-service-auth-provider
-      secrets:
-        - microservicekey-bulk-scan-sample-app-tests
-
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  devApplicationInsightsInstrumentKey: "6a512449-6a07-4994-b08d-059b9146bbe9"
```
Comment (nitinprabhu) [inline charts/bulk-scan-sample-app/values.aat.template.yaml]: we can remove this as we don't this for sample app repo

**Comment by nitinprabhu** [comment]
> i wonder if ccd api is even used here

good point @doncem don't think we use CCD so can be completely removed.

**Comment by aliveni** [comment]
> > i wonder if ccd api is even used here
> 
> good point @doncem don't think we use CCD so can be completely removed.

ccd api is not used in sample app. so removed ccd url from config

**Comment by aliveni** [comment]
> `app-insights-instrumentation-key` config is missing Please refer bulk scan processor or payment processor.

thanks, added `app-insights-instrumentation-key` config

**Pull request merged**